Apps/Common/Spalding.C

```diff
@@ -1,3 +1,18 @@
+//==============================================================================
+//!
+//! \file Spalding.C
+//!
+//! \date Jan 25 2011
+//!
+//! \author Runar Holdahl / SINTEF
+//!
+//! \brief Implementation of Spalding parametrization of a turbulent
+//!        boundary layer. Computes the mean velocity uplus parallel
+//!        to a solid wall given the distance y and the tangential
+//!        velocity component ut.
```

Comment:
This wasn't that brief... Suggest to put the second statement in the `\details` section of the class documentation only (that is, in Spalding.h). Similar change for that file also. `\brief` should be for one-liners only.